packages/supi/test/storeUsages.ts

```diff
@@ -0,0 +1,37 @@
+import prepare from '@pnpm/prepare'
+import { FindPackageUsagesResponse } from '@pnpm/store-controller-types';
+import {
+  installPkgs,
+  storeUsages,
+} from 'supi'
+import tape = require('tape');
+import promisifyTape from 'tape-promise'
+import { testDefaults } from './utils'
+
+const test = promisifyTape(tape);
```

Comment:
Plan on adding more tests here after all changes to `src` are done. Some test cases I plan on including:

- Make sure query returns all versions of a package in the store
- Make sure query returns all projects that depend on the package
- Make sure query does not return any results if package is not the store
- Make sure specifying a version number does not return other versions